composer.json

```diff
@@ -12,8 +12,8 @@
     ],
     "require": {
         "php": ">=5.3",
-        "symfony/framework-bundle": "2.*",
-        "symfony/symfony": "2.*"
+        "symfony/framework-bundle": ">=2.*",
+        "symfony/symfony": ">=2.*"
```

Comment:
This will not forbid the bundle to be installed with Symfony 4 which is not unlikely to break. I would rather use something like `^2.0|^3.0`.